src/index.js

```diff
@@ -544,11 +533,6 @@ Dropzone.propTypes = {
    */
   onDropRejected: PropTypes.func,
 
-  /**
-   * onDragStart callback
-   */
-  onDragStart: PropTypes.func,
-
```

Comment:
This is a breaking change

Reply:
It is not, from a functionality pov, because nobody can use this prop. It never gets called unless the dropzone itself is being dragged.

But syntacticly speaking, you're right, it is a breaking change.

Reply:
It is a breaking change if the api has changed.